src/Test/DocTest/Internal/Interpreter.hs

```diff
@@ -45,13 +45,14 @@ interpreterSupported = do
 --
 -- Example:
 --
--- >>> withInterpreter [] $ \i -> eval i "23 + 42"
+-- >>> withInterpreter (const (pure ())) [] $ \i -> eval i "23 + 42"
```

Comment:
Maybe you want to define `noLogger = const $ pure ()` and use it in the specs (there are tons of repetitions of `(const (pure ()))`).

Reply:
Will do